doc/compiler/jitserver/Build.md

```diff
@@ -22,159 +22,19 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
-
-cd /root \
- && wget https://api.adoptopenjdk.net/openjdk8-openj9/releases/x64_linux/latest/binary -O bootjdk8.tar.gz \
- && tar -xzf bootjdk8.tar.gz \
- && rm -f bootjdk8.tar.gz \
- && mv $(ls | grep -i jdk) bootjdk8
-
-
-export JAVA_HOME=/root/bootjdk8
-export PATH="$JAVA_HOME/bin:$PATH"
-```
-
-## VM
-
-The `JITServer` feature is not enabled by default on the `master` branch at the moment. **`--enable-jitserver` option needs to be passed in the configure step**
-
-```
-git clone https://github.com/ibmruntimes/openj9-openjdk-jdk8.git
-
-cd openj9-openjdk-jdk8
-
-bash get_source.sh -openj9-repo=https://github.com/eclipse-openj9/openj9.git -omr-repo=https://github.com/eclipse-openj9/openj9-omr.git
-
-bash configure --with-freemarker-jar=/root/freemarker.jar --with-boot-jdk=/root/bootjdk8 --enable-jitserver
-
-make all
-```
-
-Depending on where you want to fetch OpenJ9 sources from, you could use your own repository as shown below:
-
-```
-bash get_source.sh -openj9-repo=https://github.com/<Your GitHub UserID>/openj9.git -omr-repo=https://github.com/eclipse-openj9/openj9-omr.git
-```
-
-See https://www.eclipse.org/openj9/oj9_build.html for more detail.
-
-You can test the success of the build process by starting the JVM in server and client mode.
-
-```
-cd build/linux-x86_64-normal-server-release/images/jdk
-```
-
-Run:
-
-To start the JVM in server mode:
-
-```
-$./bin/jitserver
-JITServer is currently a technology preview. Its use is not yet supported
-
-JITServer is ready to accept incoming requests
-```
-
-To start the JVM in client mode:
-
-```
-$./bin/java -XX:+UseJITServer -version
-JIT: using build "Nov 6 2019 20:02:35"
-JIT level: cf3ba4120
-openjdk version "1.8.0_232-internal"
-OpenJDK Runtime Environment (build 1.8.0_232-internal-root_2019_11_06_19_57-b00)
-Eclipse OpenJ9 VM (build master-cf3ba4120, JRE 1.8.0 Linux amd64-64-Bit Compressed References 20191106_000000(JIT enabled, AOT enabled)
-OpenJ9   - cf3ba4120
-OMR      - cf9d75a4a
-JCL      - 03cb3a3cb4 based on jdk8u232-b09)
-```
-
-## JIT
-
-If you already have an SDK enabled with JITServer and just want to rebuild the JIT library, then **`export J9VM_OPT_JITSERVER=1` needs to be added to the build environment**.
-
-```
-export JAVA_HOME=/your/sdk
-export J9SRC="$JAVA_HOME"/jre/lib/amd64/compressedrefs
-export JIT_SRCBASE=/your/openj9/runtime
-export OMR_SRCBASE=/your/omr
-
-export J9VM_OPT_JITSERVER=1
-
-ln -s "$OMR_SRCBASE" "$JIT_SRCBASE"/omr
-cp "$J9SRC"/compiler/env/ut_j9jit.* "$JIT_SRCBASE"/compiler/env/
-
-cd "$JIT_SRCBASE"/runtime/compiler
-make -f compiler.mk -C "$JIT_SRCBASE"/compiler -j$(nproc) J9SRC="$JAVA_HOME"/jre/lib/amd64/compressedrefs/ JIT_SRCBASE=.. BUILD_CONFIG=debug J9_VERSION=29 JIT_OBJBASE=../objs
-cp "$JIT_SRCBASE"/compiler/../objs/libj9jit29.so "$J9SRC"
-```
-
-## DOCKER
-
-Follow the [Docker.md](Docker.md) section `build/Dockerfile` for building JITServer in Docker.
+Same instructions as [here](../../build-instructions/Build_Compiler_Only.md),
```

Comment:
I would add the work `but` before the comma: 'Same instructions as here, but to build with JITServer...`
We may also want to be more specific on what `J9VM_OPT_JITSERVER` needs to be set to.